MinPlatformPkg/PlatformInit/PlatformInitSmm/PlatformInitMm.c

```diff
@@ -0,0 +1,118 @@
+/** @file
+  This driver will register two callbacks to call fsp's notifies.
+
+  Copyright (c) 2014 - 2016, Intel Corporation. All rights reserved.<BR>
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <PiMm.h>
+
+#include "EventNotify.h"
+#include <Protocol/MmReadyToLock.h>
+#include <Protocol/MmEndOfDxe.h>
+#include <Protocol/SmmReadyToBoot.h>
+#include <Protocol/SmmExitBootServices.h>
+
+#include <Library/StandaloneMmDriverEntryPoint.h>
+#include <Library/MmServicesTableLib.h>
+#include <Library/DebugLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/BoardInitLib.h>
+#include <Library/TestPointCheckLib.h>
+#include <Library/PerformanceLib.h>
+#include <Library/HobLib.h>
+
+/**
+  Standalone MM End Of Dxe event notification handler.
```

Comment:
A lot of these functions say "Standalone MM" but this file is included for both the Traditional MM and Standalone MM drivers, right?

Reply:
Fixed to just mention MM not Standalone MM.